ProQuest.js

```diff
@@ -499,6 +517,61 @@ function getItemType(types) {
 	return guessType;
 }
 
+function scrapeEbraryResults(refs) {
+	// Since we can't chase URLs, let's get what we can from the page
+	
+	for(var i = 0; i < refs.length; i++) {
+		var ref = refs[i];
+		var hiddenData = ZU.xpathText(ref.html, './span');
+		var visibleData = ZU.xpathText(ref.html.getElementsByClassName('results_list_copy'), './text()').trim().split(';').reverse();
+		
+		var item = new Zotero.Item("book");
+
+		var date = /\(([\w\d\s]+)\)/.exec(visibleData[0]);
+		var place = /([\w,\s]+)\(/.exec(visibleData[0]);
```

Comment:
this will match the same text as above, since comma is not required, though if the place precedes date, it would match that first. Please comment on what you're parsing